.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+ci:
+    autoupdate_schedule: 'monthly'
+
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.4.3
+  hooks:
+    # Run the linter.
+    - id: ruff
+      types_or: [ python, pyi, jupyter ]
+      args: [ --fix ]
+    # Run the formatter.
+    - id: ruff-format
```

Comment:
> Thanks. Shouldn't Black also be added?

`ruff format` is a fast Black replacement.

Reply:
ah you're right, thank you.